raiden/tests/utils/transfer.py

```diff
@@ -461,12 +460,9 @@ def make_mediated_transfer(
     )
 
     if secret is not None:
-        random_sender = make_address()
+        secrethash = sha3(secret)
 
-        from_secretreveal = ReceiveSecretReveal(secret, random_sender)
-        channel.handle_receive_secretreveal(from_channel, from_secretreveal)
```

Comment:
the function `handle_receive_secretreveal` was only used from this test ... which is useless, so I remove the function and started using the `register_secret` directly